source/_integrations/discovergy.markdown

```diff
@@ -40,14 +40,117 @@ Password:
 ## Electricity meter
 
 Sensor {% term entities %} are being added for current active power usage and the all-time total consumption.
-By default, the sensors for phase-specific current active power usage are disabled.
+By default, the sensors for phase-specific current active power usage are disabled but can be enabled in the entity settings.
 
-In case you have a combined meter for consumption and production, the all-time total production is added as well.
+In case you have a bidirectional meter for consumption and production, the all-time total production is added as well.
+
+### Provided Sensors
```

Comment:
there are 2 sections called provided sensors. Could they be merged into one section?

Reply:
Yea we can merge them